Makefile

```diff
@@ -42,7 +42,8 @@ all: format build
 .PHONY: build
 build: check-git deps ## Build obsctl.
 	@echo ">> building obsctl"
-	@GOBIN=$(GOBIN) go install github.com/observatorium/obsctl
+	@GOBIN=$(GOBIN) go build -v -ldflags '-w -extldflags '-static'' -o obsctl
```

Comment:
Is there any existing issue that this addresses? 🙂 
I believe this makes it harder to test obsctl out, with having to manually add this executable to your path each time, you want to install the latest build.

We use `go install` in other CLIs too for eg in [mdox](https://github.com/bwplotka/mdox/blob/main/Makefile#L45), [bingo](https://github.com/bwplotka/bingo/blob/main/Makefile#L46) so I would prefer to keep this as a separate `make install` command if needed.

Reply:
`go install` has the side effect of replacing the binary in `$GOBIN`, which might not be what one expect when running `make build`. Building something is very different from installing, that's why we make the classic combination of `make build; make install`.

I'm happy move the `go install` to `make install`.

Reply:
+1, if I'm building a binary to test it makes sense to have it replace my path for the executable

Reply:
@moadz do you mean it makes sense that a `make build` command replaces things in your path, forever? This has such a different meaning from `build`, this sounds like `install`.